src/poliastro/maneuver.py

```diff
@@ -185,6 +185,8 @@ def lambert(cls, orbit_i, orbit_f, method=lambert_izzo, short=True, **kwargs):
 
         # Time of flight is solved by subtracting both orbit epochs
         tof = orbit_f.epoch - orbit_i.epoch
+        if tof < 0:
+            raise ValueError("Time of Flight must be positive")
```

Comment:
Can we have a more explanatory error message here? Mentioning the orbit epochs (like you initially did, but now we are on the right function)

Reply:
Sure!